app/Eloquent/Book.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Book extends Model
+{
+    use SoftDeletes;
+
+    protected $fillable = [
+    	'title',
+    	'description',
+    	'author',
+    	'publish_date',
+    	'total_pages',
+    	'avg_star',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/Book.php</br>Spaces must be used to indent lines; tabs are not allowed